examples/CMakeLists.txt

```diff
@@ -85,23 +85,39 @@ add_subdirectory(api/cpp)
 
 if(TARGET cvc5::cvc5jar)
   find_package(Java REQUIRED)
+  find_package(JNI REQUIRED)
```

Comment:
Why do we need JNI for the examples?

Reply:
We need them to run ctest java examples. Without adding the 2 directories that contain libjawt.so and libjvm.so, ctest fails with unsatisfied linking errors. 
It seems this was not needed in cvc4 because SWIG. 
```bash
/usr/lib/jvm/default-java/lib/libjawt.so;
/usr/lib/jvm/default-java/lib/server/libjvm.so
```